csi/src/findmnt.rs

```diff
@@ -0,0 +1,223 @@
+use crate::error::DeviceError;
+use serde_json::Value;
+use std::{collections::HashMap, process::Command, string::String, vec::Vec};
+
+// Keys of interest we expect to find in the JSON output generated
+/// by findmnt.
+const TARGET_KEY: &str = "target";
+const SOURCE_KEY: &str = "source";
+const FSTYPE_KEY: &str = "fstype";
+
+#[derive(Debug)]
+pub struct DeviceMount {
+    pub mount_path: String,
+    pub fstype: String,
+}
+
+#[derive(Debug)]
+struct FindmntFilter<'a> {
```

Comment:
nit; FindMntFilter but as it is confined to this file (which is good thing) Filter would work as well +)

Reply:
Done.